chainerx/_docs/routines.py

```diff
@@ -954,6 +954,27 @@ def _docs_manipulation():
 
 
 .. seealso:: :func:`numpy.swapaxes`
+""")
+
+    _docs.set_doc(
+        chainerx.repeat,
+        """repeat(a, repeats, axis=None)
+Construct an array by repeating a given array.
```

Comment:
Sorry, a real minor but per convention.
```suggestion
Constructs an array by repeating a given array.
```